setup.sh

```diff
@@ -104,3 +104,11 @@ echo "${RUNNER_TOOL_CACHE}/dart-sdk/bin" >> $GITHUB_PATH
 # Report success, and print version.
 echo -e "Successfully installed Dart SDK:"
 ${RUNNER_TOOL_CACHE}/dart-sdk/bin/dart --version
+
+# When enabled through env variables, create OIDC token for publishing on pub.dev.
+if [[ "${ACTIONS_ID_TOKEN_REQUEST_URL}" != "" && "${ACTIONS_ID_TOKEN_REQUEST_TOKEN}" != "" ]]
+then
+  PUB_TOKEN=$(curl --retry 5 --retry-connrefused -sLS "${ACTIONS_ID_TOKEN_REQUEST_URL}&audience=https://pub.dev" -H "User-Agent: actions/oidc-client" -H "Authorization: Bearer $ACTIONS_ID_TOKEN_REQUEST_TOKEN" | jq -r '.value')
+  echo "PUB_TOKEN=${PUB_TOKEN}" >> $GITHUB_ENV
```

Comment:
```suggestion
  echo "PUB_TOKEN=${PUB_TOKEN}" >> "$GITHUB_ENV"
```

Reply:
It kind of breaks the consistency of the file with the previous ones unquoted, but I don't see much issue, e.g. the others could be fixed to the new style later.